Adding backup and license files for graphics + Resolve Styling comments (#37)
DynamoDS/RefineryPrimer: 01-introduction/01-02_generative-design/01-02-03_what-goes-into-a-generative-design-process/01-02-03-01_anatomy-of-each-stage.md

## Overall process

> We can map these stages and steps together in a single diagram, this way we are able to visualize the order of each stage and their dependencies.

Reviewer: doesn't read nice as a written sentence, feels like `speech to text`.
```suggestion
We can map these stages and steps together in a single diagram, allowing us to visualize the order of each stage and their dependencies.
```
